app/javascript/src/components/Dashboard/Notes/DropdownMenu.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import { MenuVertical } from "neetoicons";
+import { Dropdown } from "neetoui";
+
+const DropdownMenu = ({ note, handleNoteDeletion }) => {
+  const handleDelete = () => {
+    handleNoteDeletion(note.id);
+  };
```

Comment:
This abstraction is pretty unnecessary because there isn't any other side effect that is happening in this block. The `handleNoteDeletion` function is already well named.